docker-compose-htsget.yml

```
@@ -0,0 +1,124 @@
+services:
+  server:
+    image: harbor.nbis.se/gdi/htsget-refserver:dev
+    container_name: htsget-server
+    command:
+      - /usr/src/app/htsget-refserver
+      - -config
+      - /config/config.json
+    networks:
+      - public
+      - secure
+    ports:
+      - 3000:3000
+    volumes:
+      #- certs:/etc/ssl/certs
+      - ./scripts/certs:/usr/local/share/ca-certificates
+      - ./config-htsget:/config
+      - ./logs:/logs
+
+  client:
+    image: python:3.11-slim
+    container_name: htsget-client
+    networks:
+      - public
+      - secure
+    command:
+      - /bin/bash
+      - -c
+      - apt update; apt install -y curl;
+        if ! which htsget >/dev/null; then pip install htsget; fi;
+        while true; do sleep 250; done
+  
+  samtools:
+    image: staphb/samtools:1.17
+    container_name: samtools-client
+    networks:
+      - public
+      - secure
+    command:
+      - /bin/bash
+      - -c 
+      - apt update; apt install -y curl jq;
+        while true; do sleep 250; done
+  
+  s3public:
+    profiles: ["demo"]
+    command: server /data --console-address ":9001"
+    container_name: s3public
+    depends_on:
+      server:
+        condition: service_started
+    environment:
+      - MINIO_ROOT_USER=access
+      - MINIO_ROOT_PASSWORD=secretkey
+      - MINIO_SERVER_URL=http://127.0.0.1:9000
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "curl",
+          "-fq",
+          "http://localhost:9000/minio/health/live"
+        ]
+      interval: 5s
+      timeout: 20s
+      retries: 3
+    image: minio/minio:RELEASE.2023-02-10T18-48-39Z
+    networks:
+      - secure
+    ports:
+      - "9003:9000"
+      - "9004:9001"
+    restart: always
+    volumes:
+      - s3pubdata:/data
+  
+  createbucket:
+    profiles: ["demo"]
+    image: minio/mc
+    container_name: mc-s3
+    depends_on:
+      s3public:
+        condition: service_started
+    networks:
+      - secure
+    entrypoint: >
+      /bin/sh -c "
+      sleep 10;
+      /usr/bin/mc -q config host add s3 http://s3public:9000 access secretkey;
+      /usr/bin/mc -q mb s3/inbox || true;
+      /usr/bin/mc anonymous set public s3/inbox;
+      /usr/bin/mc cp /data/NA12878.bam s3/inbox/NA12878.bam;
+      exit 0;
+      "
+    volumes:
+      - ./demo-data:/data
+    
+  data_downloader:
+    profiles: ["demo"]
+    image: staphb/samtools:1.17
+    container_name: data-downloader
+    depends_on:
+      createbucket:
+        condition: service_completed_successfully
+    networks:
+      - public
+      - secure
+    command:
+      - /bin/bash
+      - -c 
+      - export HTS_ALLOW_UNENCRYPTED_AUTHORIZATION_HEADER="I understand the risks";
+        export HTS_AUTH_LOCATION=token.txt;
+        echo "some-token" > token.txt;
+        apt update; apt install -y curl jq;
+        samtools view http://server:3000/reads/pub/NA12878.bam > NA12878.bam;
+       exit 0;
+    volumes:
+      - ./output:/data
+volumes:
+  s3pubdata:
```

```suggestion
volumes:
  s3pubdata:
  starter-kit-storage-and-interfaces_shared:
    external: true
```
This gives you access to the certificates from `storage-and-interfaces`